pkg/minishift/clusterup/clusterup.go

```diff
@@ -29,6 +29,7 @@ import (
 	"strings"
 	"time"
 
+	startFlags "github.com/minishift/minishift/cmd/minishift/cmd/config"
```

Comment:
it has always been config, as the package suggests, but startFlagConfig would be better as a name